woocommerce/woocommerce-ios
# Dynamic font size on connection manual screen

Closes: #5518 


###### Merge after the target branch PR

### Description

Adds support for dynamic font size on the connection instruction screen

### Testing instructions

* Go to the settings
* IPP
* Manage to card readers
* Via accessibility change font size and notice that it reflected on the instruction screen

### Screenshots


<img width="313" alt="image" src="https://user-images.githubusercontent.com/4923871/168803280-01f1878b-8173-46e4-95a7-0f1db137db36.png">
<img width="304" alt="image" src="https://user-images.githubusercontent.com/4923871/168803315-3c4c6cf9-4a0f-4e4d-be13-b52ebf9ff9eb.png">


---
